lib/datadog/appsec/security_engine/engine.rb

```diff
@@ -64,7 +60,7 @@ def add_or_update_config(config, path:)
           remove_config_at_path(DEFAULT_RULES_CONFIG_PATH) if @is_ruleset_update
 
           diagnostics = @waf_builder.add_or_update_config(config, path: path)
-          @ruleset_version = diagnostics['ruleset_version'] if diagnostics.key?('ruleset_version')
+          @builder_ruleset_version = diagnostics['ruleset_version'] if diagnostics.key?('ruleset_version')
```

Comment:
I think `builder_` prefix doesn't make it clear what is it, as I understand from the code (actually well improved now) it's a version we get after reconfiguration or update.

My suggestion is to name with "what" instead of "how" in mind. Current name reflecting how it will be used (in the builder), when we actually should say what is it, instead (suggestions: `reconfigured_ruleset_version`, `updated_ruleset_version`, `new_ruleset_version`, etc.)